src/annotator/main.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
-require('../polyfills');
+require('../shared/polyfills');
 
 var Annotator = require('annotator');
 
 // Polyfills
+
+// document.evaluate() implementation,
+// required by IE 10, 11
+//
+// This sets `window.wgxpath`
+if (!window.document.evaluate) {
```

Comment:
This polyfill has been moved here from `polyfills.js` because `document.evaluate` is only used by the annotation layer, not the sidebar app.